docs/get_started.md

```diff
@@ -69,39 +75,47 @@ docker push <your-registry>/dynamo-base:latest-vllm
 ```
 
 This documentation describes these frameworks:
-- `--framework vllm` build: see [here](examples/llm_deployment.md).
-- `--framework tensorrtllm` build: see [here](examples/trtllm.md).
+
+  - `--framework vllm` build:
+    See [LLM Deployment Examples](examples/llm_deployment.md).
+
+  - `--framework tensorrtllm` build:
+    See [TRTLLM Deployment Examples](examples/trtllm.md).
 
 After building, use this image by setting the `DYNAMO_IMAGE` environment variable to point to your built image:
+
 ```bash
 export DYNAMO_IMAGE=<your-registry>/dynamo-base:latest-vllm
```

Comment:
With the change above to consider trtllm does this need to reference the framework as well?